deadbelly/breathEZ
# Tm/resolve merge conflicts

### What does this PR do? 
>Clearly and concisely describe the feature. 

This PR fixes the merge conflicts of lost styling.

### Is this a feature or Fix/Refactor?  
Fix

### Problems Encountered & Resulting Solutions  
>State any problems encountered 
>Describe your code changes in detail for reviewers.

None.

### Where should the reviewer start?
>What file(s) and line(s) are where changes are made?

I changed all components of `css` into `scss `and imported them into out `index.scss` file. I also added back in the lost styling.

### How should this be tested?
>What steps need to be taken to confirm functionality? 

git fetch the branch
run `npm i`
run `npm start`
Look & play around with all the components

### Steps To Take
>Any questions and possible steps moving forward.

Will need to polish the styling up

### This PR is related to which issues:
>Tag issue here: `Closes #<issue_number>`
None.
